pkg/api/controller.go

```diff
@@ -56,6 +56,10 @@ const (
 	setupStateNotInitialized = "not_initialized"
 
 	DefaultMaxDeleteObjects = 1000
+
+	ResetPasswordAudience = "password_reset"
+	tokenExpiryDuration   = 5 * time.Minute
+	loginAudiance         = ""
```

Comment:
Why do we also have `loginTokenAudience` defined separately and above?  A const that is defined with different names in two locations but must have equal values is a bug waiting to happen, and is probably worse than just using a literal.

Can you export the audience values from a single location?  Possibly even create a sub-package just for that (if `import` cycles are an issue).